client/js/components/assessment_results/_assessment_result.spec.js

```diff
@@ -11,21 +11,36 @@ describe('assessment result', function() {
 
   beforeAll(function(){
     jasmine.getFixtures().fixturesPath = "base/specs_support/fixtures";
-    const settings = {};
+    const settings = {
+      assessmentKind:"summative"
+    };
     const assessment =  readFixtures("qti1/text.xml");
-    var state = {
+    const assessmentMetaData = {
+      type:"QTI1"
```

Comment:
Let's discuss the assessmentMetaData reducer I think we might be able to use a selector to get the assessment type